frontend/trpc/routes/documents/internal.ts

```
@@ -0,0 +1,124 @@
+import { TRPCError } from "@trpc/server";
+import { and, eq } from "drizzle-orm";
+import { z } from "zod";
+import { db } from "@/db";
+import { documents, documentSignatures } from "@/db/schema";
+import { baseProcedure, companyProcedure, createRouter } from "@/trpc";
+
+export const internalDocumentsRouter = createRouter({
+  // Upload PDF and create document
+  upload: companyProcedure
+    .input(z.object({
```

ESLint

[error] Insert `⏎······`

(prettier/prettier)